variables.tf

```diff
@@ -26,4 +26,10 @@ variable "aws_secret_access_key" {
 variable "aws_instance_profile_arn" {
   type = string
   description = "ARN of the AWS instance profile that will be used by CAST AI cluster-controller."
+}
+
+variable "api_url" {
+  type = string
+  description = "URL of alternative CAST AI API to be used during development or testing"
+  default = ""
```

Comment:
Why is default value of prod is not set?

Reply:
Good point. Haven't tested controller on prod, assumed we default like in agent/provider, will fix